test/onnx/test_pytorch_onnx_caffe2_quantized.py

```diff
@@ -34,7 +34,13 @@ def generic_test(self, model, sample_inputs, input_names=None, decimal=3):
         f.seek(0)
         onnx_model = onnx.load(f)
         caffe_res = c2.run_model(onnx_model, dict(zip(input_names, sample_inputs)))[0]
-        np.testing.assert_almost_equal(output.detach().numpy(), caffe_res, decimal=decimal)
+        output_diff = np.absolute(np.squeeze(output.detach().numpy()) - caffe_res)
+        max_diff = np.amax(output_diff)
+
+        # Permute pytorch output to NHWC
```

Comment:
Why is this comment here?

Reply:
It was there from where it is copy pasted from. Will remove.